### PR TITLE
opencv

### PR DESCRIPTION
**Pull request by IlyaShein**

### CODE REVIEW

File: modules/1_opencv/src/opencv_coins.cpp [hunk 1]
```diff
@@ -3,12 +3,32 @@
...
 unsigned countCoins(const Mat& img) {
-    Mat gray, thresh;
-    cvtColor(img, gray, COLOR_BGR2GRAY);
-    threshold(gray, thresh, 0, 255, THRESH_BINARY_INV + THRESH_OTSU);
-    // Remove unfilled black holes by morphological closing
-    morphologyEx(thresh, thresh, MORPH_CLOSE, Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
-
-    // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
-    CV_Error(Error::StsNotImplemented, "countCoins");
+	Mat gray, thresh;
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp:6]: Please remove redundant spaces, yours differences with old version not right

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -3,12 +3,32 @@
...
+	threshold(thresh, thresh, 0.7*maxValue, 255, 0);
+	
+
+	thresh.convertTo(thresh, CV_8U);
+	std::vector<std::vector<Point> > contours;
+	findContours(thresh, contours, RETR_EXTERNAL, CHAIN_APPROX_SIMPLE);
+	
+	int Sum = 0;
+	for (int i = 0; i < contours.size(); i++){
+	   double radius = arcLength(contours[i], true) /( 2 * CV_PI);
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp:26]: Please describe equation

File: modules/1_opencv/src/opencv_coins.cpp (hunk 1 above)
Comment (IlyaShein) [reply modules/1_opencv/src/opencv_coins.cpp:6]: In visual studio correctly displays, but here for some reason they are shifted

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (IlyaShein) [reply modules/1_opencv/src/opencv_coins.cpp:26]: I find the radius value for each contour based on the circle length formula

File: modules/1_opencv/src/opencv_coins.cpp (hunk 1 above)
Comment (allnes) [reply modules/1_opencv/src/opencv_coins.cpp:6]: please correct this

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (IlyaShein) [reply modules/1_opencv/src/opencv_coins.cpp:26]: or was it necessary to calculate the radius through the area of the contour?

File: modules/1_opencv/src/opencv_coins.cpp (hunk 1 above)
Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:6]: You can find a settings like "Replace Tab to Spaces" or "Number of whitespaces in Tab"

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:26]: The formula is ok for me

Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp:26]: `length = 2*pi*r` so `r = length / (2*pi)` approximately

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/102

**Pull request closed without merging**